README.md

```diff
@@ -1,24 +1,585 @@
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Database 
+
+<img src="images/schema.png"  width="300" height="800">
+
+### Installation
+
+1. Clone the repo
+   ```sh
+   git clone https://github.com/Tavern-Keeper-2308/tavern_keeper_be.git
+   ```
+2. Gem Bundle
+   ```sh
+    bundle
+   ```
+5. Rake
+   ```sh
+    rails db:{drop,create,migrate,seed}
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Testing
+1. Run test suite
+   ```sh
+   bundle exec rspec
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- ENDPOINTS -->
+## Endpoints
+
+### [INSERT-ENDPOINT-NAME-HERE](#INSERT-ENDPOINT-TAG_HERE)
+
+**`ENDPOINT BADGE HERE`**
+##### Example Query/Mutation (MAKE SURE TO DELETE ONE THAT IT NOT IS)
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example GraphQL Variables
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example Response
+```json
+INSERT-JSON-RESPONSE-HERE
+```
+#### Error Handling
+##### `INSERT-ERROR-CODE-HERE` INSERT-ERROR-EXPLAINATION-HERE
+##### Example Query
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example Response
+```json
+INSERT-JSON-RESPONSE-HERE
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### [Query User](#query-user)
+
+**`ENDPOINT BADGE HERE`**
+##### Example Query
+```graphql
+query getUser($id: Integer!) {
+    queryUsers (byId: $id) {
+        id
+        username
+        password
+        encounters {
+            encounter_id
+        }
+    }
+}
+```
+##### Example GraphQL Variables
+```graphql
+{
+    "id": 55
+}
+```
+##### Example Response
+```json
+{
+    "data": {
+        "queryUsers": [
+            {
+                "'id": 55,
+                "username": "Shrek",
+                "password": "password123",
+                "encounters": [
+                    {
+                        "encounter_id": 3
+                    },
+                    {
+                        "encounter_id": 5
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+#### Error Handling
+##### `INSERT-ERROR-CODE-HERE` INSERT-ERROR-EXPLAINATION-HERE
+##### Example Query
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example Response
+```json
+INSERT-JSON-RESPONSE-HERE
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### [Query Encounter](#query-encounter)
+**`ENDPOINT BADGE HERE`**
+##### Example Query
+```graphql
+query getEncounter($id: Integer!) {
+    queryEncounters (byId: $id) {
+        id
+        copper
+        silver
+        gold
+        platinum
+        electrum
+        encounter_loots {
+            item_name
+        }
+        encounter_monsters {
+            monster_name
+        }
+    }
+}
+```
+##### Example GraphQL Variables
+```graphql
+{
+    "id": 55
+}
+```
+##### Example Response
+```json
+{
+    "data": {
+        "queryEncounters": [
+            {
+                "'id": 55,
```

Comment:
Pretty sure we can remove the `'` in front of `id`. I think it's just a weird typo on the starter kit. Unless anyone else found anything different in their research?